tests/build.gradle

```diff
@@ -49,11 +48,7 @@ dependencies {
     testCompile 'io.spray:spray-json_2.11:1.3.2'
     testCompile 'com.google.code.tempus-fugit:tempus-fugit:1.2-SNAPSHOT'
     testCompile 'com.google.code.gson:gson:2.3.1'
-
-    testCompile files('$whiskhome/common/scala/build/libs/scala.jar',
-                      '$whiskhome/core/controller/build/libs/controller.jar', 
-                      '$whiskhome/core/dispatcher/build/libs/dispatcher.jar')
-    testCompile files(test_common)
+    testCompile project(':WhiskTest').sourceSets.test.output
```

Comment:
@markusthoemmes @daisy-ycguo
I need to keep the project definitions for :common:scala, :core:dispatcher and :core:controller.

If we remove them, I have got the following errors when running ./gradlew :tests:test in openwhisk-catalog:
```
openwhisk@openwhisk:~/workspace/openwhisk-catalog$ ./gradlew :tests:test
Starting a new Gradle Daemon for this build (subsequent builds will be faster).

FAILURE: Build failed with an exception.

* Where:
Build file '/home/openwhisk/workspace/openwhisk-catalog/tests/build.gradle' line: 51

* What went wrong:
A problem occurred evaluating project ':tests'.
> Could not find property 'sourceSets' on project ':WhiskTest'.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 7.614 secs
```